docs/internals/contributing/writing-documentation.txt

```diff
@@ -548,6 +548,13 @@ AdvanceCOMP's ``advpng``:
 This is based on OptiPNG version 0.7.5. Older versions may complain about the
 ``-strip all`` option being lossy.
 
+.. warning::
+
+    These commands should only be run by those who add (or modify) images in
+    their documentation changes. They are also run by Django Fellows once in a
+    while (e.g. before the final release of a new build) to optimize any images
+    that were not already optimized when they were added or modified.
```

Comment:
I'm not sure this should be a warning, since we're not really letting readers beware of a potential *issue*. It might be better as a note instead, or maybe you can just prepend it to the section, like:
```reST
If your documentation adds images to the documentation, optimize the image
compression where possible....
```

 Also, the sentence beginning 'They are also run by Django Fellows...' seems unnecessary and could be removed, as it doesn’t directly relate to the contributors' actions or guide them in any way.

Reply:
I don't have a strong opinion in the text or the type of admonition. Please create a change suggestion.